src/components/Slider/__tests__/Slider.interactions.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from "@storybook/jest";
+import { within, userEvent } from "@storybook/testing-library";
+import { delay, interactionSuite, resetFocus } from "../../../__tests__/interactions-helper";
+
+// Decrease/Increase value by mouse click on Track/Rail of Slider
+const changeSliderValueByClickingOnTrackTest = async canvas => {
+  // prepare: take sizes of slider
+  const elRail = canvas.getByTestId("monday-slider-show-value-s__rail");
+  const rect = elRail.getBoundingClientRect();
+  // waiting for render
+  await delay(300);
+  const elThumb = within(elRail).getByRole("slider");
+  // go to start
+  userEvent.click(elRail, { clientX: Math.ceil(rect.left) });
+  await delay(300);
```

Comment:
why do we need the delays (this one and the following ones) for?

Reply:
Actually I just wanted to see it at storybook and forgot to remove.
Already removing...

Reply:
Resolved